### PR TITLE
feat: add college algebra cert as upcoming change

### PR DESCRIPTION
**Pull request by moT01**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

ToDo:

### CODE REVIEW

File: client/src/templates/Introduction/components/Block.js [hunk 1]
```diff
@@ -109,11 +109,11 @@ export class Block extends Component {
     const isProjectBlock = challenges.some(challenge => {
       const isJsProject =
         challenge.order === 10 && challenge.challengeType === 5;
-
       const isOtherProject =
         challenge.challengeType === 3 ||
         challenge.challengeType === 4 ||
-        challenge.challengeType === 10;
+        (challenge.challengeType === 10 &&
+          challenge.superBlock !== 'foundational-math');
```
Comment (ShaunSHamilton) [inline client/src/templates/Introduction/components/Block.js:116]: Should we not give this a new `challengeType`? I would prefer that.

**Comment by moT01** [comment]
I think we should get this in - it adds the college algebra cert behind the upcoming flag. Hopefully, it isn't too bloated.

**Review by Sboonny — COMMENTED** [review]
`translation` and `intro` files can be deleted and you don't need to add them for the other languages, unless you are changing something in the 4 files `links`, `meta-tags`, `motiviation`, and `trending`. I may be wrong, we can ask Naomi to be sure about this.

https://contribute.freecodecamp.org/#/how-to-enable-new-languages?id=client-ui

**Review by raisedadead — APPROVED** [review]

File: tools/scripts/seed/certified-user-data.js [hunk 2]
```diff
@@ -35,6 +35,7 @@ module.exports = {
   isDataAnalysisPyCertV7: true,
   isMachineLearningPyCertV7: true,
   isRelationalDatabaseCertV8: true,
+  isCollegeAlgebraPyCertV8: false,
```
Comment (ShaunSHamilton) [inline tools/scripts/seed/certified-user-data.js:38]:
```suggestion
  isCollegeAlgebraPyCertV8: true,
```

I guess this is not forced, but we should add projects to the `completedChallenges` too.

File: curriculum/challenges/english/17-college-algebra-with-python/build-a-multi-function-calculator/multi-function-calculator.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- factor square roots
+- convert decimals to fractions and percents
+- convert fractions to decimals and percents
+- convert percents to decimals and fractions
+
+If you are struggling, you can follow the <a href="https://www.youtube.com/embed/PdsvcZNPEEs" target="_blank" rel="noopener noreferrer nofollow">video walkthrough for this project.</a>
+
+Once you're done, submit the URL to the public Colab notebook on your Google drive.
+
+## --hints--
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/17-college-algebra-with-python/build-a-multi-function-calculator/multi-function-calculator.md:29]: This should be removed, or should be `# --hints--`
```suggestion
```

File: curriculum/challenges/english/17-college-algebra-with-python/build-a-graphing-calculator/graphing-calculator.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- Shade above or below the line
+- Solve and graph a system of equations
+- Zoom in or out on a graph
+- Solve quadratic equations
+
+If you are struggling, you can follow the <a href="https://www.youtube.com/embed/EM0yNdZBdfQ" target="_blank" rel="noopener noreferrer nofollow">video walkthrough for this project.</a>
+
+Once you're done, submit the URL to the public Colab notebook on your Google drive.
+
+## --hints--
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/17-college-algebra-with-python/build-a-graphing-calculator/graphing-calculator.md:29]:
```suggestion
```

File: curriculum/challenges/english/17-college-algebra-with-python/build-a-financial-calculator/financial-calculator.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+- Estimate retirement investment balance
+- Determine how long until an amount doubles, given the rate
+- Solve logarithmic equations
+- Convert to (and from) scientific notation
+
+If you are struggling, you can follow the <a href="https://www.youtube.com/embed/c2AhGd6srJ0" target="_blank" rel="noopener noreferrer nofollow">video walkthrough for this project.</a>
+
+Once you're done, submit the URL to the public Colab notebook on your Google drive.
+
+## --hints--
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/17-college-algebra-with-python/build-a-financial-calculator/financial-calculator.md:29]:
```suggestion
```

File: curriculum/challenges/english/17-college-algebra-with-python/build-a-data-graph-explorer/data-graph-explorer.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+- Use the Pandas library to save the .csv as a dataframe
+- Print headings and the first two rows
+- Store the column names as a list
+- Choose one or two columns and convert the data to Numpy arrays
+- Display data as a scatter plot or a line graph
+- Be able to do this for different column combinations, and interpret the graphs
+
+Once you're done, submit the URL to the public Colab notebook on your Google drive.
+
+## --hints--
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/17-college-algebra-with-python/build-a-data-graph-explorer/data-graph-explorer.md:31]:
```suggestion
```

File: curriculum/challenges/english/17-college-algebra-with-python/build-three-math-games/three-math-games.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+- Algebra practice game
+  - Generate one-step and two-step problems with random integer values and the player has to input the answer
+  - Use positive and negative values. For added difficulty, make the numbers larger
+- Projectile game
+  - Display a "wall" with random height and location. Player has to move sliders to adjust a parabolic path to clear the wall
+  - For added difficulty, make a second level where players enter a, b, and c without sliders
+
+Once you're done, submit the URL to the public Colab notebook on your Google drive.
+
+## --hints--
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/17-college-algebra-with-python/build-three-math-games/three-math-games.md:30]:
```suggestion
```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/17-college-algebra-with-python/build-a-multi-function-calculator/multi-function-calculator.md (hunk 3 above)
Comment (ShaunSHamilton) [reply curriculum/challenges/english/17-college-algebra-with-python/build-a-multi-function-calculator/multi-function-calculator.md:29]: image

File: client/src/templates/Introduction/components/block.tsx [hunk 8]
```diff
@@ -332,17 +338,19 @@ class Block extends Component<BlockProps> {
     );
 
     const blockrenderer = () => {
-      if (isProjectBlock)
-        return isNewResponsiveWebDesign || isNewJsAlgos
+      if (isProjectBlock && !isOdinProject)
```
[the remark this answers is not in the document]
Comment (moT01) [reply client/src/templates/Introduction/components/block.tsx:341]: There must have been some reason for it. I don't recall at the moment, adding this made the odin project not display correctly or something maybe.

**Review by ShaunSHamilton — APPROVED** [review]
LOTM 👍

**Pull request merged**